source/content/plugins-known-issues.md

```diff
@@ -1063,6 +1063,14 @@ You can confirm usage by navigating to the Wordfence menu within your WordPress
 
 ___
 
+## WordPress Download Manager
+
+**Issue 1:** The [WordPress Download Manager](https://www.wpdownloadmanager.com/) plugin `wpdm-cache` directory may grow excessively large with generated files.
+
+**Solution:** Use an alternative download manager plugin.
```

Comment:
@whitneymeredith - Can we ask/find out what a valuable alternative could be, so that we can suggest it here?

Reply:
@michellecolon-pantheon - Jen D says there isn't a specific recommendation for an alternative at the moment.